extensions/cms/src/cmsResource/tree/cmsResourceTreeNode.ts

```diff
@@ -38,12 +39,20 @@ export class CmsResourceTreeNode extends CmsResourceTreeNodeBase {
 		try {
 			let nodes: CmsResourceTreeNodeBase[] = [];
 			if (!this.ownerUri) {
-				this._ownerUri = await this.appContext.cmsUtils.getUriForConnection(this.connection);
+				// Set back password to get ownerUri
+				if (this.connection.options.authenticationType === 'SqlLogin' && this.connection.options.savePassword === true) {
+					this.connection.options.password = await this.appContext.cmsUtils.getPassword(this.connection.options.user);
+				}
 			}
```

Comment:
But now you're not calling getUriForConnection?

Reply:
It's called underneath in createCmsServer

Reply:
Where? I'm not seeing it...

---
In reply to: [292669556](https://github.com/microsoft/azuredatastudio/pull/5989#discussion_r292669556) [](ancestors = 292669556)